platform/darwin/src/MGLAccountManager_Private.h

```diff
@@ -3,7 +3,7 @@
 @interface MGLAccountManager (Private)
 
 /// Returns the shared instance of the `MGLAccountManager` class.
-+ (instancetype)sharedManager;
+@property (class, nonatomic, readonly) MGLAccountManager *sharedManager;
```

Comment:
The use of `nonatomic` for these class properties is consistent with previous behavior when these properties were class methods. `atomic` is a remnant of cargo-cult programming that resulted in excessive locking and dispatching early on.